### ISSUE TITLE
correct loss function output

### ISSUE THREAD

**Issue by lschmiddey** [lschmiddey/deep_tabular_augmentation#14]
loss values seem to be way too high: https://www.kaggle.com/code/darkrubiks/variational-autoencoder-with-pytorch

**Comment by lschmiddey** [comment]
corrected here: https://github.com/lschmiddey/deep_tabular_augmentation/pull/15

**Comment by lschmiddey** [comment]
corrected here: https://github.com/lschmiddey/deep_tabular_augmentation/pull/15
